css/style.css

```diff
@@ -0,0 +1,21 @@
+nav {
```

Comment:
I suggest you always add a predefined style for your body element before you start inputting your main style. Try setting your body element to padding and margin of zero and box-sizing of border-box.